carma/launch/plugins.launch.py

```diff
@@ -129,6 +132,34 @@ def generate_launch_description():
                     route_following_plugin_file_path,
                     vehicle_config_param_file
                 ]
+            ),
+            ComposableNode(
+                    package='cooperative_lanechange',
+                    plugin='cooperative_lanechange::CooperativeLaneChangePlugin',
+                    name='cooperative_lanechange',
+                    extra_arguments=[
+                    {'use_intra_process_comms': True}, 
+                    {'--log-level' : GetLogLevel('cooperative_lanechange', env_log_levels) }
+                ],
```

Comment:
indentation here could be cleaned up

Reply:
Fixed here and at other locations within plugins.launch.py